cloudfoundry-operations/src/main/java/org/cloudfoundry/operations/services/Services.java

```diff
@@ -64,6 +64,13 @@
     Flux<ServiceInstance> listInstances();
 
     /**
+     * LList available offerings in the marketplace
+     *
+     * @return the service instances
+     */
+    Flux<ServiceOffering> listOfferings(GetMarketplaceRequest request);
```

Comment:
@antechrestos Nope, I'm wrong.  The change needs to be that the request type is `ListServiceOfferingsRequest`.  The rest if fine.  I'll still make the change.